integreat_cms/cms/views/mixins.py

```diff
@@ -105,3 +106,33 @@ def get_context_data(self, **kwargs):
             }
         )
         return context
+
+
+# pylint: disable=too-few-public-methods
+class SummAiContextMixin(ContextMixin):
+    """
+    This mixin provides extra context for SUMM.AI bulk action views
+    """
+
+    def get_context_data(self, **kwargs):
+        r"""
+        Returns a dictionary representing the template context
+        (see :meth:`~django.views.generic.base.ContextMixin.get_context_data`).
+
+        :param \**kwargs: The given keyword arguments
+        :type \**kwargs: dict
+
+        :return: The template context
+        :rtype: dict
+        """
+        context = super().get_context_data(**kwargs)
+        context["SUMM_AI_ENABLED"] = (
+            settings.SUMM_AI_ENABLED
+            and self.request.region.summ_ai_enabled
+            and kwargs.get("language_slug")
+            in [
+                settings.SUMM_AI_GERMAN_LANGUAGE_SLUG,
+                settings.SUMM_AI_EASY_GERMAN_LANGUAGE_SLUG,
+            ]
+        )
```

Comment:
@JoeyStk in theory, this code should make sure that the SUMM.AI option is only shown in the template when all three conditions are `True`:

1. SUMM.AI is globally enabled (aka an API key is set)
2. SUMM.AI is enabled in the current region (via the region form)
3. The current language slug is either `settings.SUMM_AI_GERMAN_LANGUAGE_SLUG` or `settings.SUMM_AI_EASY_GERMAN_LANGUAGE_SLUG`...

Do you see a potential problem with my code which causes the condition to not work on your machine?

Reply:
Please consider this to be resolved. I was able to fix the issue on my side and could test this PR properly now. Sorry for the flawed review so far. Please only consider the new review below :blush: